components/encryption/src/manager/mod.rs

```diff
@@ -1337,4 +1341,123 @@ mod tests {
             assert_eq!(buffer, content);
         }
     }
+
```

Comment:
Could these code be simplified a bit? You shouldn't need to add 8 functions for one test.

Reply:
Sure. I merged some functions, please take a look again.